_release_common.sh

```diff
@@ -53,6 +75,30 @@ function is_quarterly_release {
 	return 1
 }
 
+function prepare_branch_to_commit_from_master {
+	lc_cd "${1}"
+
+	git checkout master
+
+	git fetch "git@github.com:liferay-release/${2}.git" master
+
+	git reset --hard FETCH_HEAD
+
+	if (git branch | grep -q "${3}")
+	then
+		git branch -D "${3}"
+	fi
```

Comment:
@kiwm, can we use a parameter to force the branch deletion and not declare this `if` statement?